### ISSUE TITLE
[BUG] *Enchanter deleting items.*

### ISSUE THREAD

**Issue by FalseVII** [Minecraft-Eternal/MC-Eternal-1.12#460]
### Description of the Bug

When enchanter is running if you place more items in, it just deletes them instead of adding them to the pile currently insite.

![image](https://user-images.githubusercontent.com/74254403/158749615-5badcff8-b657-4b8e-92b8-7a5fcd6699c8.png)
![image](https://user-images.githubusercontent.com/74254403/158749646-fa39f7b4-1651-4a56-8752-7b7f1fce8e45.png)

### Expected Behaviour

Put item in, enchanter starts working.
Add another item in, it is added to the pile already in the enchanter.

### Modpack Version

1.5 Classic recipe mode

**Comment by anabsolutesloth** [comment]
Yeah unfortunately we can't fix this, most we can do is warn about it.
